src/shared/pe-binary.c

```diff
@@ -236,6 +236,5 @@ bool pe_is_uki(const PeHeader *pe_header, const IMAGE_SECTION_HEADER *sections)
 
         return
                 pe_header_find_section(pe_header, sections, ".osrel") &&
-                pe_header_find_section(pe_header, sections, ".linux") &&
-                pe_header_find_section(pe_header, sections, ".initrd");
+                pe_header_find_section(pe_header, sections, ".linux");
```

Comment:
I think we should add a comment here, something along the lines of `/* Note that the UKI spec only requires .linux, but we are stricter here, and require .osrel too, since for sd-boot it just doesn't make sense to not have that.*/`

Reply:
Okay, added.